.github/labeler.yml

```diff
@@ -0,0 +1,91 @@
+
+Building:
+- changed-files:
+  - any-glob-to-any-file: [ '.github/**',
+                            'CMakeLists.txt',
+                            'builds/**',
+                            'Makefile.am',
```

Comment:
Not sure if CMakeLists and Makefile.am should be listed here as any PR that adds new files will trigger this label.

Reply:
Technically each added file changes the building process, but I get what you mean.
Unfortunately these labels can only be changed on file basis or branch basis, so only option would be to apply building label manually then.
I think we should just live with some false positives for now and amend the config later, if needed.